docroot/sites/settings/cgov_caching.settings.php

```diff
@@ -23,22 +23,31 @@
   $config['system.performance']['js']['preprocess'] = TRUE;
   $config['system.performance']['js']['gzip'] = TRUE;
 
-  switch ($env) {
-    case 'dev':
-    case '01dev':
-    case 'int':
-    case '01test':
-    case 'test':
-    case '01live':
-    case 'ode':
-      // Cache settings.
-      $config['system.performance']['cache']['page']['max_age'] = 16588800;
-      break;
+  // MEO environments.
+  if ($_ENV['AH_ENVIRONMENT_TYPE'] === 'meo') {
+    switch ($env) {
+      case 'dev':
+      case 'stage':
+      case 'prod':
+        $config['system.performance']['cache']['page']['max_age'] = 16588800;
+        break;
+    }
+  }
+  // ACE environments.
+  else {
+    switch ($env) {
+      case 'dev':
+      case 'int':
+      case 'test':
+      case 'ode':
+        $config['system.performance']['cache']['page']['max_age'] = 16588800;
+        break;
+    }
   }
```

Comment:
Unless I'm. misreading this, do we even need the distinction? Everything is getting the same setting.